pkg/agent/proxy/proxier.go

```diff
@@ -470,25 +487,37 @@ func (p *proxier) installServices() {
 				klog.ErrorS(err, "Error when installing Endpoints flows")
 				continue
 			}
+			// mcWebhookServiceName "antrea-mc-webhook-service" is a pre-defined webhook Service for Antrea Multi-cluster
+			// instead of a Multi-cluster Service, so it should be excluded. For any Multi-cluster Service, its name will
+			// be a combination with prefix `antrea-mc-` and exported Service's name. So we need to extract the prefix to
+			// find the exported Service if there is a match.
+			if p.multiclusterEnabled && mcsLocalService != nil && svcPortName.Name != mcWebhookServiceName &&
```

Comment:
We should do this only when "mcsLocalService != nil"?

Reply:
Better to add func for "GetMCSExportedServiceName" or "GetMCSExportedServiceGroupID" to encapsulate the (tricky) logic.

Reply:
Yes, we need to check there is a matching local ClusterIP in the Endpoints first to make sure the Service is exported. So mcsLocalService won't be nil if there is a match.
Add two new functions to update `mcsLocalService`.